security/providers/oidc-common/src/test/java/io/helidon/security/providers/oidc/common/OidcConfigFromBuilderTest.java

```diff
@@ -43,4 +50,27 @@ class OidcConfigFromBuilderTest extends OidcConfigAbstractTest {
     OidcConfig getConfig() {
         return oidcConfig;
     }
+
+    @Test
+    void testCookieEncryptionPasswordFromBuilderConfig() {
```

Comment:
I do not think this is correct. Empty password is OK if the user desires so.
Can you please add a test to verify that this issue is fixed (e.g. to make sure we get the password from config)?

Reply:
I've been looking at validating the value but it is added to private vars. Will dig some more to see  if I can extract it from somewhere.

Reply:
no problem to add a package local method to retrieve the password from the builder

Reply:
OK, NW. I think I found a way to stub the config builder class so the cookie-encryption-password value can be retrieved and validated. Working on it.